### PR TITLE
Update spec for lvs_noheadings

### PR DESCRIPTION
**Pull request by JoySnow**

### CODE REVIEW

**Comment by esakaiev** [comment]
test this please

File: uploader.json [hunk 1]
```diff
@@ -4389,8 +4389,8 @@
         "lvs_noheadings": {
             "host": [
                 {
-                    "archive_file_name": "/insights_commands/lvs_--nameprefixes_--noheadings_--separator_-a_-o_lv_name_lv_size_lv_attr_mirror_log_vg_name_devices_region_size_data_percent_metadata_percent_--config_global_locking_type_0",
-                    "command": "/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent --config=\"global{locking_type=0}\"",
+                    "archive_file_name": "/insights_commands/lvs_--nameprefixes_--noheadings_--separator_-a_-o_lv_name_lv_size_lv_attr_mirror_log_vg_name_devices_region_size_data_percent_metadata_percent_segtype_--config_global_locking_type_0",
+                    "command": "/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent,segtype --config=\"global{locking_type=0}\"",
```
Comment (bfahr) [inline uploader.json]: @JoySnow you will need to remove these changes, this portion of the json is not currently used.

**Comment by YakovLittle** [comment]
test this please

**Comment by JoySnow** [comment]
@bfahr 
I am not sure if I should add it as well.

Both `lvs_noheadings_all` and `lvs_noheadings` exist in insights-core.
Since there is no `lvs_noheadings_all` in the uploader.json, I guess there must be some reason for this.

@xiangce Any idea about this?

**Comment by xiangce** [comment]
@bfahr @JoySnow - the `lvs_noheadings_all` is added in [commit-71c95b87e](https://github.com/RedHatInsights/insights-core/commit/71c95b87e) by @lonicerae early this year. In the same PR, the other two specs `pvs_noheadings_all` and `vgs_noheadings_all` were also added together with the `lvs_noheadings_all` which are used to display all the lvm devices. They are essential for related parsers and plugins. Unfortunately, we didn't have the `CM processes for Spec change` at that time and they were not added to the `uploader.json`.

They should be added, but I think better add them via another respective PR. @lonicerae would you please to open JIRA request and asserts PR to make up the 3 `_noheadings_all` specs?

BTW, @bfahr the case of `_noheadings` and `_noheadings_all` specs is very similar to the `ss-tulpn` (l=listening) and `ss-tupna`(a=all=listening+non-listening). I think `_noheadings_all` must also cover the `_noheadings`. If there is a way to distinguish the filters lvm devices (seems it not possible yet 😭), we can replace the `_noheadings` with the `_noheadings_all`  😈

**Comment by bfahr** [comment]
This PR is approved.

**Comment by bfahr** [comment]
This core PR has already been merged.  This PR is approved and can be merged.

**Pull request merged**